pkg/tests/ossm/smcp_tls_ssl_test.go

```diff
@@ -43,16 +43,16 @@ func TestTLSVersionSMCP(t *testing.T) {
 		ns := "bookinfo"
 		t.Cleanup(func() {
 			oc.RecreateNamespace(t, ns)
-			oc.WaitSMCPReady(t, meshNamespace, smcpName)
+			oc.RecreateNamespace(t, meshNamespace)
 			oc.ApplyString(t, meshNamespace, smcpName)
 			t.LogStep("Enable the Service Mesh Control Plane mTLS to true")
-			oc.MergePatch(t, meshNamespace,
+			oc.Patch(t, meshNamespace,
```

Comment:
This patch is not needed inside the cleanup, if you are creating again the namespace this will be deleted, if needed for the test they should be outside the cleanup